cmd/snap/cmd_changes_test.go

```diff
@@ -155,6 +155,39 @@ Do +2016-04-21T01:02:03Z +2016-04-21T01:02:04Z +some summary
 	c.Assert(err, check.ErrorMatches, `no changes of type "foobar" found`)
 }
 
+func (s *SnapSuite) TestTasksLastQuestionmark(c *check.C) {
+	n := 0
+	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
+		c.Check(r.Method, check.Equals, "GET")
+		c.Assert(r.URL.Path, check.Equals, "/v2/changes")
+		switch n {
+		case 0, 1:
+			fmt.Fprintln(w, `{"type": "sync", "result": []}`)
+		case 2, 3:
+			fmt.Fprintln(w, mockChangesJSON)
+		default:
+			c.Errorf("expected 4 calls, now on %d", n)
+		}
+		n++
+	})
+	for i := 0; i < 2; i++ {
+		rest, err := snap.Parser().ParseArgs([]string{"tasks", "--last=foobar?"})
```

Comment:
Given that this also works for watch and abort and changes, Should we test those as well? test can probably/hopefully trivially be reused. This ensures we do not regress if we forget to check for noChangeFoundOk in the users of the mixin.